include/aspect/material_model/interface.h

```diff
@@ -824,6 +835,20 @@ namespace aspect
                                   const SymmetricTensor<2,dim> &strain_rate,
                                   const Point<dim>             &position) const=0;
 
+        /*
+         * Return the anisotropic viscosity tensor of the model as a function
+         * of temperature, pressure, composition, strain rate, and position.
+         *
+         * @note As in the viscosity calculation, the strain rate given is
+         * computed without the $\frac 13 \nabla \cdot \mathbf u \mathbf 1$
+         * term as is required for consistency in the compressible case.
+         */
+        virtual SymmetricTensor<4,dim> viscosity_tensor (const double                  temperature,
```

Comment:
Ok. Is that because using the InterfaceCompatibility class is depreciated, or for another reason?